src/sentry/static/sentry/app/actionCreators/health.jsx

```diff
@@ -1,18 +1,29 @@
 const BASE_URL = org => `/organizations/${org.slug}/health/`;
 
+// Gets the period to query with if we need to double the initial period in order
+// to get data for the previous period
+const getPeriod = (originalPeriod, shouldDoublePeriod) => {
+  if (!shouldDoublePeriod) return originalPeriod;
```

Comment:
Are leading 0's problematic in this case? When the value is doubled `parseInt()` will strip off any leading 0s.